src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBadGenomicKmers.java

```diff
@@ -0,0 +1,135 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.SparkProgramGroup;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import scala.Tuple2;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * SparkTool to identify 63-mers in the reference that occur more than 3 times.
+ *
+ * Created by tsharpe on 12/4/15.
+ */
+@CommandLineProgramProperties(summary="Use spark to find the set of high copy number kmers in a reference.",
+        oneLineSummary="find ref kmers with high copy number",
+        programGroup = SparkProgramGroup.class)
+public final class FindBadGenomicKmers extends GATKSparkTool
```

Comment:
Our current convention is to have spark enabled tools end in Spark.  i.e. `FindBadGenomicKmersSpark`